.github/workflows/release.yml

```diff
@@ -0,0 +1,43 @@
+name: Release
+
+on:
```

Comment:
wouldn't you want this to depend on the Test workflow?

Reply:
this creates a PR that we can merge to trigger a release, so we wouldn't do that if tests are failing